consistency/tool/comparison/comparison.py

```diff
@@ -1,42 +1,37 @@
 from strsimpy.sorensen_dice import SorensenDice
-import sys
+from google.cloud.apigeeregistry.applications.v1alpha1.consistency import (
+    consistency_report_pb2 as cr,
+)
 
 
 class Comparison:
-    def __init__(self, stub, new_words):
+    def __init__(self, stub, new_words, word_groups=None, noise_words_index = None):
         self.stub = stub
         self.new_words = new_words
-
-    def get_word_group_ids(self):
-        # TODO: retrieve the WordGroups we have stored from registry.
-        # return them in an appropriate format for easy comparison
-        return []
+        self.word_groups=word_groups
+        self.noise_words_index  = noise_words_index 
```

Comment:
Why are we accepting `noise_words_index` and not the "NOISE" WordGroup that we store in the registry?